test/Confluent.Kafka.IntegrationTests/Tests/Tests.cs

```diff
@@ -30,13 +30,19 @@ public static IEnumerable<object[]> KafkaParameters()
         {
             if (kafkaParameters == null)
             {
-                var codeBase = typeof(Tests).GetTypeInfo().Assembly.CodeBase;
```

Comment:
`Assembly.Location` provides the full path of the tests dll. And it's exactly what is needed here. 

From the [MSDN article](https://msdn.microsoft.com/en-us/library/system.reflection.assembly.codebase(v=vs.110).aspx) for the `CodeBase` property:

> To get the absolute path to the loaded manifest-containing file, use the `Assembly.Location` property instead.

I can rollback this and move to anther PR. Should I do it?